### DISCUSSION TITLE
Doesn't work for me

### DISCUSSION THREAD

**Discussion by Deadite** [valinet/ExplorerPatcher#112] [General]
I get a notification on starting Windows 11:  Unable to find symbols for OS version 10.0.22000.258.

I've installed the latest version of ExplorerPatcher, but the previous version did the same.

Using Windows 11 Pro UK, which was upgraded from 10.  At first I thought it might be Bitdefender but I added an exception for this file so not sure what's causing it.

**Comment by valinet** [comment]
Well, at first run, it has to download some data from Microsoft. Unless it does that, it won’t function. Make sure nothing blocks or interferes with that. You should see network activity after that message (in Task Manager for example) and then after some time when the download finishes, Explorer would restart and that’d be it, you’d see the old taskbar and from there on it starts working.

**Reply by Deadite** [reply]
Yes you were, you were correct - the culprit was Bitdefender firewall blocking the download. I turned it off, rebooted and it works...

Thanks!

**Reply by valinet** [reply]
Yeah, it also a bit cumbersome for the user as I do not throw any notification saying, "oh, well, I can't do my job" from my side. Maybe that should be something I'd have to consider in a subsequent update. It's also a bit weird that Bitdefender blocks File Explorer from accessing the network if you ask me (the patcher runs in the context of the `explorer.exe` process). Anyway, I am glad that you got it sorted out!

**Reply by valinet** [reply]
And also I appreciate that you read the rules in issues. Should this have been a bug on my side, I would have migrated it to issues no problem, but since it was a bit of a misunderstanding only, it lived here just fine. Thanks!

**Comment by sanigerson** [comment]
Hi, I'm thinking, i'm having some kind of same problem.
I install dll under windows folder, restart explorer.exe, but nothing happens. Also, %appdata%\ExplorerPatcher folder doesn't seems to be created but if i want to delete dll file under windows folder, windows doesn't let me to do it. Any suggestion?
I deactivated AV, also connected to internet.
Thanks a lot.

Windows OS: 22000.318

**Reply by BraINstinct0** [reply]
You cannot directly delete it; first rename it, restart explorer, then delete it.
If same issues occur, then it may be a bug - in that case, you may open an issue(but I'd prefer replying back here).
However as %appdata%\ExplorerPatcher isn't created, it means that it probably isn't doing anything.

**Comment by exmateriae** [comment]
Before anything, thanks for your great program!
My issue is that pretty much every week explorerpatcher seems to crash and I get an empty taskbar : only the windows button and the quick settings (wifi, sound, etc) appear.
After that, I have the same problem as described above. Sometimes killing and restarting the explorer will make it work again and sometimes I try dozens of times with no results... (today is one of these days)
I have looked in the network activity but I am unable to understand which program is blocking the download. I see "your phone" app, adobe creative cloud and onedrive use the network but that's mainly it...

Thanks in advance

**Reply by valinet** [reply]
Thanks

Idk, 2/3 times I think I experienced that as well during development as well, where the taskbar will not load fully when explorer restarts. A restart if explorer again always solves it. It is very rare, I don’t know how to trigger that, so almost impossible to catch. What more can I do? May be some race condition, may be something else, I don’t really know. I didn’t code the taskbar or the Windows shell, it is what it is, once I find and can confirm and isolate some issue, I am quick to fixing it, other than that, idk. All this madness should not been needed in the first place, this is not a long term solution if the OS vendor continuously fights its users.

I don’t understand exactly what you are experiencing, but that doesn’t matter that much either, for me to understand. I can’t do much without a clear pattern for reproducing something, and even then, who knows, I am not a wizard, there are some issues I am yet to know how to solve. If you want to investigate, idk, enable the console, attach with a debugger, things like these. It’s the kind of utility that it is, apparently there always will be issues with it, it’s a nightmare when you can’t control or know exactly how the software works, but you try to do so much with it. This is the curse of EP.

Thank you

**Reply by exmateriae** [reply]
Thank you for your very fast answer! I'll update this if I find a consistent solution to the problem

**Reply by exmateriae** [reply]
I seem to have found the issue : it is with Windows 11 itself. There is a bug of Iris Service (the system that loads tips for instance) that makes the taskbar empty in Windows 11. I think it was this that was blocking the whole thing.

You can disable it with this command and then just reboot : 
`reg delete HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\IrisService /f`

Hope this helps anyone with the same problem and thanks again valinet for your awesome explorerpatcher 👍

**Reply by valinet** [reply]
Thank you very much for posting your findings. Good to know and indeed, maybe it helps others as well.
